src/exo/LoopIR_compiler.py

```diff
@@ -434,17 +434,17 @@ def from_lines(x):
 // Compiler feature macros adapted from Hedley (public domain)
 // https://github.com/nemequ/hedley
 
-#if defined(__has_builtin)
```

Comment:
I am confused. Why were these changed? These are referring to C builtins, not Exo.